pkg/interactor/data.go

```diff
@@ -27,8 +27,9 @@ func (d *DataInteractor) GetAll(order, skip, take int, startDate, finishDate tim
 	data, err := d.DataStore.Get(selectOrder, skip, take, startDate, finishDate)
 	if err != nil {
 		d.logger.Error(err)
+		return nil, err
 	}
-	return data, err
+	return data, nil
```

Comment:
Please, fix this in another commit or mention this change explicitly in the commit message.

Reply:
Done!